CONTRIBUTORS.md

```diff
@@ -2426,4 +2426,9 @@ Noida
 #### Name: [Ítalo Epifânio](https://github.com/itepifanio)
 - Place: Natal, Brazil
 - Bio: Web developer PHP and Python
-- Github: [Ítalo Epifânio](https://github.com/itepifanio)
\ No newline at end of file
+- Github: [Ítalo Epifânio](https://github.com/itepifanio)
```

Comment:
I'm not sure why it sees this as modified.  Was it caused by the new line to the file? Should I have made the changes at the top?